cloud-computing/src/app.py

```diff
@@ -64,26 +64,27 @@ def handle_app_developer_secrets():
     This function demonstrates how to use application-developer secrets in an
     iExec application.
     The following environment variables are used:
-        - IEXEC_APP_DEVELOPER_SECRET_0: first secret set by the application
+        - IEXEC_APP_DEVELOPER_SECRET_1: first secret set by the application
```

Comment:
Are you sure we should use `IEXEC_APP_DEVELOPER_SECRET_1`? As far as I understand the way we're going, having multiple app developer secrets won't be implemented before long. Shouldn't we only use `IEXEC_APP_DEVELOPER_SECRET` so that it's easier for people?